### DISCUSSION TITLE
Configure file extensions?

### DISCUSSION THREAD

**Discussion by thoni56** [manoelcampos/asciidoctor-ghpages-action#32] [Q&A] [answered]
Is there a way to configure which file extensions this action will recognize? I often use `.asciidoc` which is not captured and converted.

**Comment by manoelcampos** [comment, marked as answer]
Use `adoc_file_ext` param. Check all params at the [example workflow](https://github.com/manoelcampos/asciidoctor-ghpages-action/blob/main/.github/workflows/asciidoctor-ghpages.yml).

**Comment by thoni56** [comment]
Aah! That's what the comments at the end of that file means...

I also found `action.yml` which gives some clues ;-)
